alacritty.yml

```diff
@@ -66,7 +66,10 @@ window:
   #title: Alacritty
 
   # Window class (Linux only):
-  #class: Alacritty
+  #class: [Alacritty, Alacritty]
+
+  # GTK theme variants (Linux only):
+  #gtk_theme_variant: None
```

Comment:
These options should both be uncommented by default if possible. That way users can just directly change them without having to uncomment them first.

I also think it might be worthwhile to describe `gtk_theme_variant` a bit more, since I'm not familiar with it myself. Is this just Dark/Light? If so, then this should be an enum, not a string.

Reply:
Only dark/light tend to be used, but GTK apparently allows arbitrary strings:

https://developer.gnome.org/gtk3/stable/GtkCssProvider.html#gtk-css-provider-get-named